### PR TITLE
[zh-cn]: create doc for HTMLParamElement

### PR DESCRIPTION
**Pull request by fuchunhui**

### CODE REVIEW

File: files/zh-cn/web/api/htmlparamelement/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: HTMLParamElement
+slug: Web/API/HTMLParamElement
+l10n:
+  sourceCommit: 7e6e5f5d4c1c984af3d4ebf9399042d19eccea1e
+---
+
+{{APIRef("HTML DOM")}}{{Deprecated_Header}}
+
+**`HTMLParamElement`** 接口提供用于操作 {{HTMLElement("param")}} 元素的特定属性（除了常规 {{domxref("HTMLElement")}} 接口之外，它还可以通过继承来使用），表示一对键和一个充当 {{HTMLElement("object")}} 元素的参数值。
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlparamelement/index.md:10]: 不是很确定，仅供参考

```suggestion
**`HTMLParamElement`** 接口提供用于操作 {{HTMLElement("param")}} 元素（表示一对充当 {{HTMLElement("object")}} 元素的参数的键和值）的特定属性（除了常规 {{domxref("HTMLElement")}} 接口之外，它还可以通过继承来使用）。
```

File: files/zh-cn/web/api/htmlparamelement/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+{{InheritanceDiagram}}
+
+## 实例属性
+
+_从其父接口 {{domxref("HTMLElement")}} 继承属性。_
+
+- {{domxref("HTMLParamElement.name")}} {{Deprecated_Inline}}
+  - : 表示参数名称的字符串。它反映 [`name`](/zh-CN/docs/Web/HTML/Element/param#name) 属性。
+- {{domxref("HTMLParamElement.value")}} {{Deprecated_Inline}}
+  - : 表示与参数关联值的字符串。它反映 [`value`](/zh-CN/docs/Web/HTML/Element/param#value) 属性。
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlparamelement/index.md:21]:
```suggestion
  - : 表示与参数关联的值的字符串。它反映 [`value`](/zh-CN/docs/Web/HTML/Element/param#value) 属性。
```

File: files/zh-cn/web/api/htmlparamelement/index.md (hunk 1 above)
Comment (fuchunhui) [reply files/zh-cn/web/api/htmlparamelement/index.md:10]: https://github.com/mdn/content/blob/fcbe69e21d85708248ff60021b1937817844f51c/files/en-us/web/api/htmlparamelement/index.md?plain=1#L12C204-L12C244

Comment (yin1999) [reply files/zh-cn/web/api/htmlparamelement/index.md:10]: 可以看一下文档 https://developer.mozilla.org/en-US/docs/Web/HTML/Element/param

**Review by yin1999 — APPROVED** [review]
Thanks @fuchunhui

**Pull request merged**